h5py/_hl/files.py

```diff
@@ -348,6 +351,22 @@ def __init__(self, name, mode=None, driver=None,
         track_order
             Track dataset/group/attribute creation order under root group
             if True. If None use global default h5.get_config().track_order.
+        strategy
+            The file space handling strategy to be used.  Only allowed when
+            creating a new file (mode w, w- or x).  Defined as:
+            h5f.FSPACE_STRATEGY_FSM_AGGR = 0  FSM, Aggregators, VFD
+            h5f.FSPACE_STRATEGY_PAGE = 1      Paged FSM, VFD
+            h5f.FSPACE_STRATEGY_AGGR = 2      Aggregators, VFD
```

Comment:
We don't generally use low-level API values as parameters for the high-level API. In keeping with how we do e.g. file drivers, the high-level API would probably accept short strings (e.g. `'page'` or `'aggregate'`).